core-bundle/contao/templates/_new/content_element/list.html.twig

```diff
@@ -6,5 +6,9 @@
 {% endblock %}
 
 {% block list_item %}
-    {%- if as_editor_view %}{{ item|raw }}{% else %}{{ item|insert_tag|raw }}{% endif -%}
+    {%- if as_editor_view %}
+        {{ item|raw }}
+    {% else %}
+        {{ item|insert_tag|raw }}
+    {% endif -%}
 {% endblock %}
```

Comment:
I think when we structure it like this, we need

```twig
    {% if as_editor_view %}
        {{- item|raw -}}
    {% else %}
        {{- item|insert_tag|raw -}}
    {% endif %}
```

but I am not sure.

/cc @m-vo @ausi

Reply:
We want to remove all whitespace, but in your version @fritzmg the whitespace before `{% if` and after `endif %}` is not removed. I’d write it like so:

```twig
    {%- if as_editor_view -%}
        {{ item|raw }}
    {%- else -%}
        {{ item|insert_tag|raw }}
    {%- endif -%}
```

Reply:
Changed in 8bd552f5952b80a01c9ef721233a31191f98e8f0.